### PR TITLE
fix: tag/workspace scheduler switch

### PR DESCRIPTION
**Pull request by iutx**
#### What type of this PR

Add one of the following kinds:
/kind bug

#### What this PR does / why we need it:
upgrade screenshoot about testing result:
image

before:
image

after:
image

#### Which issue(s) this PR fixes:

- Fixes #your-issue_number
- [Erda Cloud Issue Link](paste your link here)

#### Specified Reviewers:

/assign @luobily 

#### ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English |   Fix scheduler with tag/workspace      |
| 🇨🇳 中文    |   修复了平台标签调度开关与 etcd 不一致问题           |

#### Need cherry-pick to release versions?

Add comment like `/cherry-pick release/1.0` when this PR is merged.

> For details on the cherry pick process, see the [cherry pick requests](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md#how-to-cherry-pick-a-merged-pr) section under [CONTRIBUTING.md](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: .erda/migrations/cmdb/20211102-cluster-scheduler-cover-tag.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+def entry():
+    """
+    please implement this and add it to the list entries
+    """
+
+    condition_clusters = CoClusters.objects.filter(Q(type='k8s') | Q(type='edas'))
+
+    for cluster in condition_clusters:
+        fmt_scheduler = json.loads(cluster.scheduler)
```
Comment (dspo) [inline .erda/migrations/cmdb/20211102-cluster-scheduler-cover-tag.py:48]: if the `cluster.scheduler` is empty, it will cause panic

Comment (iutx) [reply .erda/migrations/cmdb/20211102-cluster-scheduler-cover-tag.py]: That does not currently exist.

**Pull request merged**